src/features/surveys/layout/SurveyLayout.tsx

```diff
@@ -120,7 +133,12 @@ const SurveyLayout: React.FC<SurveyLayoutProps> = ({
       defaultTab="/"
       ellipsisMenuItems={[
         {
-          label: messages.layout.actions.delete(),
+          label: <>messages.layout.actions.move()</>,
+          onSelect: () => setIsMoveDialogOpen(true),
+          startIcon: <ArrowForward />,
+        },
+        {
+          label: <>messages.layout.actions.delete()</>,
```

Comment:
I understand where the inspiration to add the fragments came from, but they are not neccessary :), and adding them to the delete label is outside of the task